File: src/main/java/szewek/flux/recipe/WashingRecipe.java
package szewek.flux.recipe;

import net.minecraft.item.ItemStack;
import net.minecraft.util.ResourceLocation;
import szewek.flux.F;

public final class WashingRecipe extends AbstractMachineRecipe {
	public WashingRecipe(ResourceLocation idIn, MachineRecipeSerializer.Builder builder) {
		super(F.R.WASHING, idIn, builder);
	}

	@Deprecated
	public WashingRecipe(ResourceLocation idIn, String group, MachineRecipeSerializer.Builder builder) {
		this(idIn, builder.withGroup(group));
	}

	@Override
	public ItemStack getToastSymbol() {
		return new ItemStack(F.B.WASHER);
	}


}

File: src/main/java/szewek/flux/recipe/WashingRecipe.kt
package szewek.flux.recipe

import net.minecraft.item.ItemStack
import net.minecraft.item.crafting.IRecipeSerializer
import net.minecraft.util.ResourceLocation
import szewek.flux.FBlocks
import szewek.flux.FRecipes

class WashingRecipe(idIn: ResourceLocation, groupIn: String, builder: MachineRecipeSerializer.Builder)
    : AbstractMachineRecipe(FRecipes.WASHING, idIn, groupIn, builder) {
    override fun getIcon() = ItemStack(FBlocks.WASHER)

    override fun getSerializer(): IRecipeSerializer<*> = FRecipes.WASHING_SERIALIZER
}